.github/actions/action-maven-release/action.yml

```
@@ -53,6 +45,6 @@ runs:
     - name: "Upload Artifact"
       uses: actions/upload-artifact@v4
       with:
-        name: target
+        name: ${{inputs.app-path}}
```

```suggestion
        name: "${{inputs.app-path}}-mvn"
```
Probably needed to not collision with node build